.gitignore

```diff
@@ -2,6 +2,7 @@
 /target
 /.settings/**
 .classpath
+.java-version
```

Comment:
Just curios, what is '.java-version'? I can't say that I've ever seen the file.

Otherwise, I'm okay with the change.

Reply:
.java-version is an artifact of the jenv tool, which allows you to manage/switch Java versions per directory. I use it all the time and it's awesome. You can trivially switch between Java 7, 8, 11, 16 and retest to make sure you are achieving your compatibility goals, etc.